docs/v1/user-story

```diff
@@ -0,0 +1,124 @@
+was quickly and non-disruptively upgraded.  (OK, I skipped the part where I tested the darn thing for a week - but you get 
+the idea)  That’s just amazing.  Again, kudos to whoever is working that update service broker over at the database vendor!
+
+13. Discover unapproved update to a service’s resources  
+While I was on a week long vacation, the interns got a call about some customer issue and they worked it for a couple days 
+and determined that the container size needed to be changed.  And they just changed it on their own since they did not want 
+to bother me on vacation.  The service definition I made clearly did not make that a parameter they could change, though and 
+they didn’t think to tell me about it when I got back since everything went back to normal by then.  But, I look at my 
+messages and there’s this alert telling me that the service instance that’s handling all our support communications is out 
+of compliance because of this setting.  I was initially upset about this, but when I looked at it in more detail, I 
+discovered that the interns were spot on.  After a certain point of load (which will not stop going up), we needed to start 
+using bigger containers and not just keep scaling out more and more tiny ones.  (Not only did the immediate problem go away, 
+but paradoxically, the cost went down which was nice - they solved another problem I’m sure I would have had next quarter)  
+But, now we need to roll this out more deliberately so that the setting sticks and that new instances are deployed properly 
+as well.  It’s a good thing I got the notification or I’d have never known.  Thanks Kubernetes!
+
+14. Update type, create update template, and redeploy service 
+So, I do for my service what the the database people did for me.  I created a new version, tested it out, and deployed it to 
+the catalog (without any marketplace in the way, since this is an internal app).  Now, I’m getting messages about my chat 
+app for tech support being out of date and the tech support people are calling me asking me if I knew that the app had a new 
+version (well, yeah, I know, I created it :-).  I guess those notifications really work…  But, now I’m the guy who needs to 
+build the update service broker and I’m worried about that.  It’s not what the database vendor went through, thankfully, 
+because the chat app is stateless.  One of the default update service brokers that come with kubernetes should just work 
+with minor modifications.  I choose between blue-green (where traffic is distributed between the versions), deploying a new 
+version entirely and having a cutover, and per customer traffic splitting.  Per customer won’t work because I really only 
+have one customer (the support app itself) and full cutover seems risky no matter how much testing I do.  So, I select the 
+blue/green update option and give it the arguments that tell it how fast to ramp from 10% to 100% on the new version.  
```

Comment:
I should know more about how this interacts with deployments.  Maybe the broker here just uses the deployments APIs for rollouts.  I can work with some of the engineers to make a more explicit story here.